src/Core/Form/IdentifiableObject/DataHandler/SqlRequestFormDataHandler.php

```diff
@@ -41,19 +44,36 @@ final class SqlRequestFormDataHandler implements FormDataHandlerInterface
      */
     private $commandBus;
 
+    /**
+     * @var SqlRequestFormDataValidator
+     */
+    private $requestSqlFormDataValidator;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
     /**
      * @param CommandBusInterface $bus
      */
-    public function __construct(CommandBusInterface $bus)
-    {
+    public function __construct(
+        CommandBusInterface $bus,
+        SqlRequestFormDataValidator $requestSqlFormDataValidator,
+        TranslatorInterface $translator
+    ) {
         $this->commandBus = $bus;
+        $this->requestSqlFormDataValidator = $requestSqlFormDataValidator;
```

Comment:
If I am not wrong, this change in the constructor is a BC Break.

This class is `final` but if someone creates this class manually
```
$myService = new SqlRequestFormDataHandler($bus);
```
Then following this PR, his code will crash?

Reply:
The fact is, we don't consider final class as BC for months :sweat: 
A lot of BC breaks have forgotten because of this :(